### DISCUSSION TITLE
Number of assets suddenly becomes 1/1 upon loading projects?

### DISCUSSION THREAD

**Discussion by TheGithubUS** [ScratchAddons/ScratchAddons#7454] [Help]
*I already posted this on the Scratch forums, but someone said it is a "browser extension feature". I'm just gonna think that this is a Scratch Addons problem (since that's the only Scratch-related extension i have installed)
Apparently this has been a problem for a few days actually. This problem seems to only happen on “Add yourself singing” (or AYS) projects.
As i said earlier, someone said that it is a feature from a browser extension, and if it is a Scratch Addons feature, i would appreciate it if anyone knows which feature is causing this.
ignore this: My browser / operating system: Windows NT 10.0, Chrome 124.0.0.0, Flash 32.0 (release 0)

**Comment by Samq64** [comment]
Do you have the custom default project addon enabled? Also are you running 1.37 or 1.38?

**Reply by TheGithubUS** [reply]
I don't have custom default project addon enabled and i'm running on v1.38.0

**Reply by Samq64** [reply]
Is it just the progress bar or the actual project?

**Reply by TheGithubUS** [reply]
It's the actual project. Although the sprites and code still loads perfectly fine.

**Comment by BroJac5246** [comment]
> This problem seems to only happen on “Add yourself singing” (or AYS) projects.

Can you provide a link to an example project where this bug is happening?

**Reply by TheGithubUS** [reply]
https://scratch.mit.edu/projects/1016658012/ I don't know if the bug is happening for you.

**Reply by BroJac5246** [reply]
Hmm it worked correctly for me, showing X/143 assets loaded, even on v1.38.0 on Chrome 124.

**Comment by WorldLanguages** [comment]
This might be related to #7169.
